tests/e2e/cucumber/environment/index.ts

```diff
@@ -25,11 +25,19 @@ import {
   keycloakCreatedUser
 } from '../../support/store'
 import { Group, User } from '../../support/types'
-import { getTokenFromLogin, setAccessToken } from '../../support/utils/tokenHelper'
+import { setAccessToken } from '../../support/utils/tokenHelper'
 import { createdTokenStore, keycloakTokenStore } from '../../support/store/token'
 import { removeTempUploadDirectory } from '../../support/utils/runtimeFs'
-import { getAccessToken, refreshToken, setupKeycloakAdminUser } from '../../support/api/keycloak'
+import {
+  setAccessTokenForKeycloakUser,
+  setupKeycloakAdminUser,
+  refreshAccessTokenForKeycloakUser
+} from '../../support/api/keycloak'
```

Comment:
you should be able to do the import from this path if you add the new file in https://github.com/owncloud/web/blob/1dac55db71273ade86ba13767a706b65110de121/tests/e2e/support/api/keycloak/index.ts#L1-L2
```suggestion
import {
  setAccessTokenForKeycloakUser,
  setupKeycloakAdminUser,
  refreshAccessTokenForKeycloakUser,
  refreshAccessTokenForKeycloakOcisUser,
  setAccessTokenForKeycloakOcisUser
} from '../../support/api/keycloak'
```